editor.planx.uk/src/ui/editor/SettingsSection.tsx

```diff
@@ -3,14 +3,7 @@ import { styled } from "@mui/material/styles";
 import { contentFlowSpacing } from "@planx/components/shared/Preview/Card";
 import React, { ReactNode } from "react";
 
-interface RootProps extends BoxProps {
-  background?: boolean;
-  "data-testid"?: string;
-}
-
-const Root = styled(Box, {
-  shouldForwardProp: (prop) => prop !== "background",
-})<RootProps>(({ background, theme }) => ({
+const Root = styled(Box)(({ bgcolor, theme }) => ({
```

Comment:
I had to revert `bgcolor` for `background` since this was acting as a boolean to conditionally style the `<SettingsSection>` component